tests/common/test_cgroups.py

```diff
@@ -304,6 +321,133 @@ def test_limits(self):
         self.assert_limits(ext_name="normal_extension", expected_cpu_limit=40, exception_raised=True)
 
 
+class TestCGroupsTelemetry(AgentTestCase):
```

Comment:
The current tests cover poll and collect... we need tests for the rest of the methods